Tests/ML/utils/test_model_util.py

```diff
@@ -74,9 +70,6 @@ def test_create_model_from_lightning_checkpoint(test_output_dirs: OutputFolderFo
         assert config._train_output_size is None
     # method to get all devices of a model
     loaded_model = load_from_checkpoint_and_adjust_for_inference(config, checkpoint_path)
-    # Information about epoch and global step must be present in the message that on_checkpoint_load writes
-    assert str(FIXED_EPOCH) in loaded_model.checkpoint_loading_message
```

Comment:
what about checking that default epoch is still in the loading message? it must be 0 if no training has been done?
If the intent here is to check that the loading_message comprises the epoch retrieved from the checkpoint file regardless of current_epoch becoming a protected property that can't be changed outside the fit_loop. 
This can catch potential bugs in checkpoints.

Reply:
Excellent point, have added this!